Imaginea/AdaptiveCards
# Extended the Background Image property support to the container elements

Worked to resolve the issue: https://github.com/Imaginea/AdaptiveCards/issues/72

Steps to test the PR:

1. Launch the App.
2. Navigate to "Container.BackgroundImageRepeatHorizontally.json" from the visualiser.
3. Expected: The backgroundImage will repeat horizontally in the container.
As same as above, we can test RepeatVertically.
